src/SprykerEco/Zed/UnzerApi/Dependency/External/Guzzle/UnzerApiToGuzzleHttpClientAdapterInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerApi\Dependency\External\Guzzle;
+
+use SprykerEco\Zed\UnzerApi\Dependency\External\Guzzle\Response\UnzerApiGuzzleResponseInterface;
+
+interface UnzerApiToGuzzleHttpClientAdapterInterface
```

Comment:
```suggestion
interface UnzerApiToHttpClientAdapterInterface
```

According to RFC https://spryker.atlassian.net/wiki/spaces/RFC/pages/18350081/ACCEPTED+RFC+Adapters+to+vendor+classes